domains/aaron.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "AaronPerezPerez",
+    "email": "aarperper@gmail.com"
+  },
+  "record": {
+    "NS": "terminal-portfolio-h80h9v8ub-aaronperezperez.vercel.app"
```

Comment:
This should be a CNAME not NS where you typed it.

Reply:
mb, fixing it now